server/prisma/schema.prisma

```diff
@@ -16,32 +16,36 @@ model Session {
-  imagePath        String?
-  isAssigned       Boolean                  @default(false)
-  classCode        String?                  @db.VarChar(45)
-  codeValidUntil   DateTime?
-  isPublic         Boolean                  @default(false)
-  isDeleted        Boolean                  @default(false)
-  sortIndex        BigInt
-  licenseCode      String?                  @default("CCDUAL") @db.VarChar(10)
-  owner            users                    @relation(fields: [ownerId], references: [userId], onDelete: NoAction, onUpdate: NoAction)
-  parentFolder     content?                 @relation("folderStructure", fields: [parentFolderId], references: [id], onDelete: NoAction, onUpdate: NoAction)
-  subFolders       content[]                @relation("folderStructure")
-  documents        documents[]
-  license          licenses?                @relation(fields: [licenseCode], references: [code], onDelete: NoAction, onUpdate: NoAction)
-  contentFeatures  contentFeatures[]
-  assignmentScores assignmentScores[]
-  promotedContent  promotedContent[]
-  sharedWith       contentShares[]
-  classifications  contentClassifications[]
-  contentViews     contentViews[]
+  id                    Bytes                    @id @default(dbgenerated("(uuid_to_bin(uuid(),1))")) @db.Binary(16)
+  ownerId               Bytes                    @db.Binary(16)
+  isFolder              Boolean
+  parentFolderId        Bytes?                   @db.Binary(16)
+  name                  String
+  createdAt             DateTime                 @default(now())
+  lastEdited            DateTime                 @default(now())
+  imagePath             String?
+  isAssigned            Boolean                  @default(false)
+  classCode             String?                  @db.VarChar(45)
+  codeValidUntil        DateTime?
+  isPublic              Boolean                  @default(false)
+  isDeleted             Boolean                  @default(false)
+  sortIndex             BigInt
+  licenseCode           String?                  @default("CCDUAL") @db.VarChar(10)
+  owner                 users                    @relation(fields: [ownerId], references: [userId], onDelete: NoAction, onUpdate: NoAction)
+  parentFolder          content?                 @relation("folderStructure", fields: [parentFolderId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  subFolders            content[]                @relation("folderStructure")
+  documents             documents[]
+  license               licenses?                @relation(fields: [licenseCode], references: [code], onDelete: NoAction, onUpdate: NoAction)
+  contentFeatures       contentFeatures[]
+  assignmentScores      assignmentScores[]
+  promotedContent       promotedContent[]
+  sharedWith            contentShares[]
+  classifications       contentClassifications[]
+  contentViews          contentViews[]
+  librarySource         libraryActivityInfos?    @relation("source")
```

Comment:
If I understand this correct, if `content` is the source for a library activity, then `librarySource` points to the activity info for which this record is the source. The name make me think it is pointing to the source. I can't think of a good name, but one option would be to swap the names `librarySource` and `libraryActivity` in this table, and same for the following two fields.

Bad names I thought of instead of [`librarySource`, `libraryActivity`]: [`libraryActivityFromSource`, `librarySourceForActivity`], [`libraryActivityFromThisSource`, `librarySourceForThisActivity`],